### ISSUE TITLE
Cannot read property 'theme' of undefined

### ISSUE THREAD

**Issue by zywudev** [ele828/hexo-prism-plugin#14]
the installation information

`$ npm i -S hexo-prism-plugin
hexo-site@0.0.0 D:\hexo\blog
`-- hexo-prism-plugin@2.0.0
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"ia32"})
`
when I executed **hexo clean**, an error occurred. 

`**ERROR Plugin load failed: hexo-prism-plugin
TypeError: Cannot read property 'theme' of undefined**
    at D:\hexo\blog\node_modules\hexo-prism-plugin\index.js:23:48
    at D:\hexo\blog\node_modules\hexo\lib\hexo\index.js:229:12
    at tryCatcher (D:\hexo\blog\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (D:\hexo\blog\node_modules\bluebird\js\release\promise.js:510:31)
`

**Comment by ele828** [comment]
This is because you didn't specify configuration in your config file.

And this is also a bug for the plugin itself, I will fix it later. But the temporary solution would be adding the followings to your config file.

Thanks for letting me know.

```
prism_plugin:
  mode: 'preprocess'    # realtime/preprocess
  theme: 'default'
  line_number: false    # default false
```

**Comment by ele828** [comment]
Fixed #21
